pkg/products/observability/prometheusRules.go

```diff
@@ -355,7 +355,7 @@ func (r *Reconciler) newAlertsReconciler(logger l.Logger, installType string) re
 						"sop_url": resources.SopApiManagementTenantCRFailed,
 						"message": "An APIManagementTenant CR has failed to reconcile. See the labels for details.",
 					},
-					Expr:   intstr.FromString(`tenants_summary{provisioningStatus!="3scale account ready"}`),
+					Expr:   intstr.FromString(`tenants_summary{provisioningStatus!="3scale account ready" && provisioningStatus != "won't provision"}`),
```

Comment:
I think this suggestion will make the `ci/prow/multitenant-rhoam-e2e` test pass. CC: @Stellar-9 
```suggestion
					Expr:   intstr.FromString(`tenants_summary{provisioningStatus!="3scale account ready"} and tenants_summary{provisioningStatus != "won't provision"}`),
```